filters/auth/oidc.go

```diff
@@ -795,6 +848,10 @@ func (f *tokenOidcFilter) tokenClaims(ctx filters.FilterContext, oauth2Token *oa
 		return nil, "", requestErrorf("claims do not contain sub")
 	}
 
+	if err = f.handleDistributedClaims(r.Context(), idToken, oauth2Token, tokenMap); err != nil {
+		log.Error(err)
```

Comment:
Why don't we fail here on error? IMO in the auth related code it should be strict

Reply:
ok